app/controllers/pensions/unauthorisedPayments/SurchargeAmountController.scala

```diff
@@ -44,7 +45,7 @@ class SurchargeAmountController @Inject()(authAction: AuthorisedAction,
 
   val amountForm: Form[BigDecimal] = AmountForm.amountForm(
     emptyFieldKey = "unauthorisedPayments.surchargeAmount.error.noEntry",
-    wrongFormatKey = "unauthorisedPayments.surchargeAmount.error.incorrectFormat"
+    wrongFormatKey = "common.error.incorrectFormat"
```

Comment:
Why was this changed? Personally, I'd rather have "scoped" messages even when duplicated. And it seems to be the more consistent pattern in this application.

Reply:
@salman-a-dev explained to me that there's an issue having duplicate messages (note: not message keys).

Please see [this spike](https://erasmos-solutions-limited.atlassian.net/browse/SUB-126).